### PR TITLE
MDEXP-79- Adding HrID in job execution

### PR DESCRIPTION
**Pull request by KVupp**
## Purpose
https://issues.folio.org/browse/MDEXP-79 - To add Hr Id sequence when a job execution is created

## Approach
- create a sequence for HR ID
- use the nextval in sequence to add HRID while creating a new job execution entry

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/test/java/org/folio/rest/impl/FileUploadServiceTest.java [hunk 1]
```diff
@@ -131,6 +132,7 @@ public void shouldUploadFile_return200Status(TestContext context) throws IOExcep
     File uploadedFile = new File(uploadedFileDefinition.getSourcePath());
     assertTrue(FileUtils.contentEquals(fileToUpload, uploadedFile));
     assertEquals(uploadedFileDefinition.getJobExecutionId(),  jobExecutions.getJobExecutions().get(0).getId());
+    assertNotNull(jobExecutions.getJobExecutions().get(0).getHrId());
```
Comment (Igor-Gorchakov) [inline src/test/java/org/folio/rest/impl/FileUploadServiceTest.java]: Maybe it makes sense later to create the endpoint to get job item by id, to do not to retrieve the collection

**Comment by Igor-Gorchakov** [comment]
Kruthi have you tested it locally? It works fine?

**Comment by KVupp** [comment]
> Kruthi have you tested it locally? It works fine?

yes, I added a validation to the existing test cases too

File: src/test/java/org/folio/rest/impl/FileUploadServiceTest.java (hunk 1 above)
Comment (KVupp) [reply src/test/java/org/folio/rest/impl/FileUploadServiceTest.java]: that is true. We'll have to create more endpoints in the future too

**Pull request merged**